src/PyCall.jl

```diff
@@ -708,6 +708,14 @@ for (mime, method) in ((MIME"text/html", "_repr_html_"),
 end
 
 #########################################################################
+# Expose Python docstrings to Julia doc system
+
+if isdefined(Docs, :getdoc)  # Introduced in Julia .6
+    Docs.getdoc(o::PyObject) = Markdown.parse(o[:__doc__])
```

Comment:
Just returning a string doesn't seem to look too great since newlines are printed literally:

```
help?> pandas.Series
"\n    One-dimensional ndarray with axis labels (including time series).\n\n    Labels need not be unique but must be any hashable type. The object\n    supports both integer- and label-based indexing and provides a host of\n    methods for performing operations involving the index. Statistical\n    methods from ndarray have been overridden to automatically exclude\n    missing data (currently represented as NaN)\n\n    Operations between Series (+, -, /, *, **) align values based on their\n    associated index values-- they need not be the same length. The result\n    index will be the sorted union of the two indexes.\n\n    Parameters\n    ----------\n    data : array-like, dict, or scalar value\n        Contains data stored in Series\n    index : array-like or Index (1d)\n        Values must be unique and hashable, same length as data. Index\n        object (or other iterable of same length as data) Will default to\n        RangeIndex(len(data)) if not provided. If both a dict and index\n        sequence are used, the index will override the keys found in the\n        dict.\n    dtype : numpy.dtype or None\n        If None, dtype will be inferred\n    copy : boolean, default False\n        Copy input data\n    "
```